lib/assets/javascripts/twine.js.coffee

```diff
@@ -312,6 +312,7 @@ for attribute in ['placeholder', 'checked', 'disabled', 'href', 'title', 'readOn
   setupAttributeBinding(attribute, attribute)
 
 setupAttributeBinding('innerHTML', 'unsafe-html')
+setupAttributeBinding('innerText', 'inner-text')
```

Comment:
If we do this, it should just be added to the list above, making the binding `bind-innerText`.